src/styles/index.css

```diff
@@ -0,0 +1,248 @@
+body {
+  margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use either single or double quotes consistently throughout the CSS. Here, 'Times New Roman' uses single quotes, while "Roboto" doesn't have any. Consider using one style for consistency.